### ISSUE TITLE
Announce: The new maintainer

### ISSUE THREAD

**Issue by achimnol** [aio-libs/aiomonitor#333]
As I have presented [aiomonitor-ng (next generation)](https://github.com/achimnol/aiomonitor-ng), which is a fork of this project, as a session talk and poster in PyCon US 2023, @webknjaz has promoted me as the maintainer of this project.

I'm going to do the following things during/after the PyCon US 2023 sprints:

- [ ] Backport the changes of aiomonitor-ng
- [ ] Rename the `master` branch to `main`
- Revamp the CI/CD and fix the broken test suite (let it use `pytest-asyncio`)
- Add a minimal web UI support in addition to the terminal UI
- Diagnose an issue which causes negative side-effects to the application event loop when the telnet server is broken (more details will follow as a separate issue)
- Integrate [pystack](https://github.com/bloomberg/pystack) as an additional option to inspect the deeper side of task stacks

Sprinters and contributors are also welcome!

**Comment by webknjaz** [comment]
@wbarnha ^

**Comment by webknjaz** [comment]
I could but I don't know what the side effects would be. Can we do it after the PRs?

**Comment by achimnol** [comment]
> I could but I don't know what the side effects would be. Can we do it after the PRs?

GitHub takes care of retargeting the PRs, but if you're worried, it's okay to handle this after #328.

**Comment by wbarnha** [comment]
Congratulations @achimnol! I saw your fork a while back and I'm glad you're getting the attention you deserve. https://github.com/aio-libs/aiomonitor/pull/328 adds a pipeline that's supposed to be the standard for existing aio-libs projects.

**Comment by webknjaz** [comment]
@wbarnha hopefully, you'll be able to migrate to a standard pipeline setup?

**Comment by wbarnha** [comment]
Yup! Thanks for the support, I'll make sure everything is in order.
